docs/terraform-provider-build.md

```diff
@@ -0,0 +1,31 @@
+# Steps to build terraform providers
+
+Follow below steps to setup terraform providers. These steps are required to be followed before running the automation.
+
```

Comment:
A note to follow below sections depending on the client arch would help.

Reply:
Updated documentation to emphasize the steps required to be followed based on Terraform Client Machine Architecture.